src/App.js

```diff
@@ -1,23 +1,24 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { Home } from "./components/home/Home";
+import React from "react";
+import { Routes, Route } from "react-router-dom";
+import { History } from "./components/history/History";
+import { VideoPage } from "./components/videopage/VideoPage";
+import { WatchLater } from "./components/watchlater/WatchLater";
+import { LikedVideos } from "./components/likedvideos/LikeVideosPage";
+import { PlayList } from "./components/playlist/PlayList";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Routes>
```

Comment:
You can create a separate file for defining all your routes, as your app grows maintaining that file would be easy and it will also help in refactoring your code.